app/constants/constants.js

```diff
@@ -2,6 +2,9 @@
 /* eslint-disable no-multi-str */
 import {concat} from 'ramda';
 
+// Default to 1 week for `refreshInterval` for backwards compatability
```

Comment:
for backwards-compatibility?

Reply:
Oh, I've just got it. This is to prevent old versions of falcon from running new scheduled queries as `setInterval(runQuery, NaN)`. Could you add something like that to comment, please?

Reply:
I believe this is to address my concerns about what happens if an non-cron-supporting version of Falcon is started in an environment where there are persisted queries that have been created with a cronInterval.